src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -33,6 +33,13 @@ public static JsonDocument CreatePaginationConnectionFromJsonElement(JsonElement
             // Maintains the connection JSON object *Connection
             Dictionary<string, object> connectionJson = new();
 
+            // in dw we wrap array with "" and hence jsonValueKing is string instead of array.
+            if (root.ValueKind == JsonValueKind.String)
```

Comment:
```suggestion
            if (root.ValueKind is JsonValueKind.String)
```

nit: prefer `is` to `==`